composer.json

```diff
@@ -11,18 +11,19 @@
     ],
     "require": {
         "ext-json": "*",
+        "ext-zip": "*",
         "keboola/gelf-server": "^2.0",
+        "keboola/input-mapping": "^12.2",
         "keboola/oauth-v2-php-client": "^2.2",
         "keboola/object-encryptor": "^0.3",
-        "keboola/storage-api-php-client-branch-wrapper": "^1.0.0",
         "keboola/output-mapping": "^14.0",
-        "keboola/input-mapping": "^12.2",
         "keboola/php-utils": "^2.0",
+        "keboola/storage-api-php-client-branch-wrapper": "^1.0.0",
         "keboola/syrup": "^11.3",
+        "keboola/workspace-provider": "^1.0",
         "mustache/mustache": "^2.13",
         "symfony/validator": "^v2.8.52|^v4.4.11",
-        "vkartaviy/retry": "^v0.2.2",
-        "ext-zip": "*"
+        "vkartaviy/retry": "^v0.2.2"
```

Comment:
not relevant here, but we should probably switch to our fork of this eventually https://github.com/keboola/retry

Reply:
yes, but not relevant here :D

Reply:
testy pohoda.. I have removed all the related tests :D